applications/asset_tracker_v2/src/modules/Kconfig.debug_module

```diff
@@ -0,0 +1,48 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menuconfig DEBUG_MODULE
+	bool "Debug module"
+
+if DEBUG_MODULE
+
+config DEBUG_MODULE_MEMFAULT_USE_EXTERNAL_TRANSPORT
+	bool "Use external transport when sending memfault data"
+	help
+	  If enabled, the debug module will forward captured Memfault data via events of type
+	  DEBUG_EVT_MEMFAULT_DATA_READY. The data should be forwarded via the
+	  configured cloud to Memfault. E.g via AWS IoT, Azure IoT Hub, or nRF Cloud.
+	  If this option is disabled, the debug module will use Memfault firwmare SDKs own internal
+	  HTTP transport. Transporting Memfault data over an already pre-established
+	  transport can save data associated with maintaining multiple connections.
+
+config DEBUG_MODULE_MEMFAULT_HEARTBEAT_INTERVAL_SEC
+	int "Memfault heartbeat interval in seconds"
+	default 1800
+	help
+	  Configuration that sets how often Memfault captures and prepares data that will be sent
+	  to Memfault cloud. Reduce this interval to get more frequent updates to Memfault.
+	  Note that data transmitted to Memfault cloud is still dependent on how often the
+	  application sends data, which can be manipulated by setting the application's real-time
+	  configurations.
+
+endif
```

Comment:
This `endif` should probably be at the end of the file

Reply:
I removed the if guard for these statements as `DEBUG_MODULE_MEMFAULT_CHUNK_SIZE_MAX` is used in `debug_module_event.h` which is included in the `cloud_module`. Either we define a fixed size buffer in `debug_module_memfault_data` (which must be rather big) or we need to have `DEBUG_MODULE_MEMFAULT_CHUNK_SIZE_MAX` always exposed.